### PR TITLE
Add new routes and update the introduced base page

### PR DESCRIPTION
**Pull request by Saphs**

### CODE REVIEW

File: src/app/components/lobby/news/news.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div>
+  <h1>Hier kommen dann alle news zum spiel hin</h1>
```
Comment (DanielLiebler) [inline src/app/components/lobby/news/news.component.html:2]: *Spiel, *News

File: src/app/components/lobby/login/login.component.html [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
+      </label>
+    </div>
+    <div class="buttons">
+      <button (click)="onLogin()">{{languageObjects.logInSubmit}}</button>
+      <span></span>
+      <span (click)="openRegisterDialog()" class="register-link">{{languageObjects.logInRegister}} <i
+        class="fas fa-angle-double-right"></i></span>
+    </div>
+    <div class="error-text"><i *ngIf="errorMessage.length > 0" class="fas fa-exclamation-triangle"></i><span
+      *ngIf="errorMessage.length > 0" [innerHTML]="errorMessage"></span></div>
```
Comment (DanielLiebler) [inline src/app/components/lobby/login/login.component.html:26]: why [innerHTML]

**Review by DanielLiebler — DISMISSED** [review]

File: src/app/components/lobby/news/news.component.html (hunk 1 above)
Comment (Saphs) [reply src/app/components/lobby/news/news.component.html:2]: Love

File: src/app/components/lobby/login/login.component.html (hunk 2 above)
Comment (Saphs) [reply src/app/components/lobby/login/login.component.html:26]: Love (wont fix)

Comment (DanielLiebler) [reply src/app/components/lobby/login/login.component.html:26]: @manuelstolze

**Review by DanielLiebler — APPROVED** [review]

**Pull request merged**